analyzer/testdata/src/extra/file.go

```diff
@@ -241,7 +241,7 @@ func stringsContains() {
 
 func fmtFprintf(x int) {
 	os.Stderr.WriteString(fmt.Sprintf("foo: %d", x))  // want `\Qsuggestion: fmt.Fprintf(os.Stderr, "foo: %d", x)`
-	os.Stderr.WriteString(fmt.Sprintf("message"))     // want `\Qsuggestion: fmt.Fprintf(os.Stderr, "message")`
+	os.Stderr.WriteString("message")                  // want `\Qsuggestion: fmt.Fprintf(os.Stderr, "message")`
```

Comment:
This is a file for unit test purpose. I.e any file under the testdata/src directory is intentionally written with issues that are supposed to be caught by the analyzer.
You can see the code comment states ` want `\Qsuggestion: fmt.Fprintf(os.Stderr, "message")`` because the static analyzer unit test is supposed to create an issue.
If you actually change that message, then the unit test will no longer be able to make a proposal to replace `fmt.Sprintf("message")` with the simpler `"message"`, and the unit test will fail.